lfs/scanner_git_test.go

```diff
@@ -0,0 +1,76 @@
+package lfs_test // to avoid import cycles
+
+// This is for doing complete git-level tests using test utils
+// Needs to be a separate file from scanner_test so that we can use a diff package
+// which avoids import cycles with testutils
+
+import (
+	"testing"
+
+	. "github.com/github/git-lfs/lfs"
```

Comment:
They don't really need to use it there, either, they're only using two things from `strings` :wink: [Effective Go](https://golang.org/doc/effective_go.html#package-names) prefers to avoid it, but I really only noticed it because some tooling doesn't handle it well (e.g. `godef`).